### ISSUE TITLE
Tweets don't open in the timeline

### ISSUE THREAD

**Issue by ZusorCode** [ZusorCode/GoodTwitterChrome#5]
Tweets stopped opening in the timeline. They now jump to the author page. I can't investigate this right now as I am on vacation.

**Comment by mnoqlo** [comment]
It was fixed by logging out and logging back in.

**Comment by Aurrius** [comment]
> It was fixed by logging out and logging back in.

That worked for me, thank you!

**Comment by millemj** [comment]
Worked here, too, although I had a hard time finding a logout link. Just added it to the URL (twitter.com/logout) and that worked.

Great job, Zusor, enjoy your vacation!

**Comment by filipre** [comment]
Worked for me as well :) You can logout if you click on your profile icon on the upper right

<img width="213" alt="Screenshot 2019-08-01 at 00 13 25" src="https://user-images.githubusercontent.com/8502198/62251836-4ead9600-b3f1-11e9-9b3d-5f5481b0bab2.png">

**Comment by millemj** [comment]
Since I'm an old f*rt, I have that page zoomed to 175%, so that particular pop-up menu ended at "Keyboard Shortcuts" and wouldn't scroll with the page. After reading your reply I unzoomed and there it was! Thanks!

**Comment by rite2ace** [comment]
Had the same issue, logout and login worked :D

**Comment by asadkhilji** [comment]
> It was fixed by logging out and logging back in.

It worked for me. Thanks

**Comment by amigokin** [comment]
> It was fixed by logging out and logging back in.

works like a charm! thank you!

**Comment by S4WA** [comment]
it worked, thanks
> It was fixed by logging out and logging back in.

**Comment by mazal** [comment]
> Tweets stopped opening in the timeline. They now jump to the author page. I can't investigate this right now as I am on vacation.

I went to your Troubleshooting page for this issue, and followed the "delete cookies" and log back in instruction—worked great! Btw, I failed to find you on twitter.. odd that you don't have a twitter presence? LOL. Anyway, I want to thank you for not only a terrific product, but a well thought out, and elegant fix it link right in the GoodTwitter widget.

Kudos,
Mazal
